src/routes/Login.tsx

```diff
@@ -28,7 +28,7 @@ export const LoginRoute: FC = () => {
 
   return (
     <div className="flex flex-col items-center justify-center p-4">
-      <Logo className="h-16 w-16" isDark />
+      <LogoIcon className="h-16 w-16" isDark />
```

Comment:
Might as well do the same as with the other icons in #1274! Also, using `size-` over `w-` and `h-`

Reply:
This is our Gitify logo which doesnt have a size property...